Ready-To-Use-UI-Demo/Droid/MainActivity.cs

```diff
@@ -353,8 +356,8 @@ protected override void OnActivityResult(int requestCode, [GeneratedEnum] Result
                     // Estimate blur of imported barcode
                     // Estimating blur on already cropped barcodes should
                     // normally yield the best results, as there is little empty space
-                    var estimator = new IO.Scanbot.Sdk.ScanbotSDK(this).CreateBlurEstimator();
-                    fragment.Blur = estimator.EstimateInBitmap(bitmap, 0);
+                    var estimator = new IO.Scanbot.Sdk.ScanbotSDK(this).CreateDocumentQualityAnalyzer();
+                    fragment.Blur = estimator.AnalyzeInBitmap(bitmap, 0)?.Ordinal() ?? 0;
```

Comment:
On `BarcodeDialogFragement`, we have `public float Blur { get; internal set; } = -1;`

I believe this should be updated to the enum value and then also any references to the text "blur" should be replaced with either "quality" or "detected quality". We should avoid any conversions from an enum value to a float in this context because the ordinal value has no meaning on its own.